enterprise/server/test/integration/remote_execution/remote_execution_test.go

```diff
@@ -1812,16 +1812,19 @@ func TestActionMerging_Hedging(t *testing.T) {
 	rbe.AddBuildBuddyServer()
 	rbe.AddExecutor(t)
 
-	// This script runs quickly and outputs "FAST" if `fname` exists.
-	// Otherwise it runs slowly and outputs "SLOW".
+	// This script runs quickly and outputs "FAST" if `fname` exists. Otherwise
+	// it writes a line to `counter`, runs slowly, and then outputs "SLOW".
 	fname := fmt.Sprintf("/tmp/%s", uuid.New().String())
+	counter := fmt.Sprintf("/tmp/%s", uuid.New().String())
 	flakyScript := fmt.Sprintf(`FILE="%s"
+COUNTER="%s"
 if [ -f $FILE ]; then
-    echo "FAST"
+	echo "FAST"
```

Comment:
nit: Mind using four spaces instead of a tab for consistency with the rest of this file?